src/lib/room.js

```diff
@@ -38,6 +38,11 @@ export const onChatClose = async () => {
 		postChatUrl: null,
 		room: null,
 	});
+	if (isWebView()) {
+		// Hide Print
+		store.setState({ hidePrint: true });
+		parentCall('hidePrint');
+	}
```

Comment:
@AlexanderKanakis This is just setting `hidePrint` flag when the widget is running in the webview. But I think what we want to achieve is when `hidePrint` is true we want to not show the `Save Transcript` button. Is that correct @ear-dev?

Reply:
yes, that is correct @Shailesh351